app/main/posts/modify/location.html

```diff
@@ -1,15 +1,55 @@
-<form ng-submit="searchLocation()">
-    <div class="input-location">
-        <div class="input-inline">
-            <input name="{{name}}" type="text" placeholder="{{'location.placeholder' | translate}}" ng-model="searchLocationTerm" ng-disabled="processing">
-            <button class="btn btn-info" type="submit" ng-disabled="processing">
-                <div class="loading" ng-show="processing"><div class="line"></div><div class="line"></div><div class="line"></div></div>
-                <span translate>location.search<span>
-            </button>
-
-            <button class="button-secondary alt" type="button" ng-click="clear()" ng-disabled="processing" translate>location.clear</button>
+ 
+<div class="location-wrapper">
+    <div id="map" class="map" style="height: 265px; width: 100%;"></div>
```

Comment:
`id` here still needs to be unique. We can have multiple location editors on a page.